prodigy/documentation/biweekly_reports/PR_1.md

```diff
@@ -0,0 +1,38 @@
+# Biweekly progress Report 1 (08.03.22)
```

Comment:
We don't need these files on GitHub. You can keep them in the local repo if you like. In the GitHub we should only have final documentation.

Reply:
- [X] The prodigy wheel file should also not be uploaded! You need to pay to get this file!

- [X] Did you make sure, that only the facts can be annotated by the annotators? Yes that is why I used the blocks task with an integrated span manual task for the facts and a simple html display for the meta info.

-[X] Did you include a link to the original court decision for the annotators?
- [ ] I think the judgment outcome should be clearly visible for the annotators. 
- [X] The data and "work in progress docu" should not be on GitHub, this should be in the .gitignore.
- [ ] Please follow the python style guide (e.g. dataset_creation instead of datasetCreation).